src/lib/flight_tasks/tasks/AutoMapper/FlightTaskAutoMapper.cpp

```diff
@@ -194,13 +189,11 @@ float FlightTaskAutoMapper::_getLandSpeed()
 		const float land_speed = math::gradual(_dist_to_ground,
 						       _param_mpc_land_alt2.get(), _param_mpc_land_alt1.get(),
 						       _param_mpc_land_speed.get(), _constraints.speed_down);
-		const float head_room = _constraints.speed_down - land_speed;
-
-		speed = land_speed + 2 * (0.5f - throttle) * head_room;
 
-		// Allow minimum assisted land speed to be half of parameter
-		if (speed < land_speed * 0.5f) {
-			speed = land_speed * 0.5f;
+		// user input assisted land speed
+		if (_param_mpc_land_rc_help.get()) {
+			_sticks.evaluateSticks(_time_stamp_current);
+			speed = land_speed + _sticks.getPositionExpo()(2) * land_speed;
```

Comment:
If the behavioral change is intended, don't forget to update the description of the parameter [MPC_LAND_RC_HELP](https://dev.px4.io/master/en/advanced/parameter_reference.html#MPC_LAND_RC_HELP)

Reply:
I just realized that the change makes is potentially quite a bit slower depending on the configuration. Let's preserve the possibilities.

Reply:
I'm not sure if we really want to enable such a high downwards speed close to ground. I refactored the implementation to be more clear and adjusted the parameter description. It's the last commit (the rest is unchanged).